krm-functions/configinject-fn/fn/function.go

```diff
@@ -139,84 +140,118 @@ func (f *FnR) desiredOwnedResourceList(forObj *fn.KubeObject) (fn.KubeObjects, e
 	}
 
 	resources := fn.KubeObjects{}
-	// walk through all the package revisions and check if the dependent resources are ready
+	// walk through all the package revisions and build a map of the pr(s) that
+	// have the packagename and are in a repo that has deployment true
+	// The map will contain the latest published revision of the pr, if no pr
+	// is published it will have a reference to this pr
 	// we assume there needs to be 1 dependency that resolves
-	found := false
+	prmap := map[string]porchv1alpha1.PackageRevision{}
```

Comment:
Should this be a map of pointers and we do DeepCopy, instead of a map of structs?

Reply:
ok fixed it